src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -26,7 +26,8 @@ case class ClusterRecord(id: Long,
                          initBucket: String,
                          machineConfig: MachineConfigRecord,
                          serviceAccountInfo: ServiceAccountInfoRecord,
-                         stagingBucket: Option[String]
+                         stagingBucket: Option[String],
+                         dateAccessed: Timestamp
```

Comment:
According to the changelog [above](https://github.com/DataBiosphere/leonardo/pull/374/files#diff-f927344c0e0828b21c2d481787449924R6), `dateAccessed` is `nullable`. Shouldn't the line above be
```scala
dateAccessed: Option[Timestamp]
```
then? Same for `case class Cluster` in `LeonardoModel`.

Reply:
I think it might be better to have the dateAccessed field be non nullable with the default value of of the current timestamp